index.html

```diff
@@ -415,9 +415,13 @@
       <a href="/you-may-like"
         >You May Also Like</a
       >
-      <a href="/news-articles" target="_blank"
+      <!-- <a href="/news-articles" target="_blank"

```

Comment:
Well, don't remove this for now. As I said, I will consider removing this later on. But, this can go below on the nav-bar maybe after Rate Song?